Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -14,43 +14,14 @@ Note that we'll use the [OWASP Mobile Testing Guide Crackmes](https://github.com
 
 Make sure that the following is installed on your system:
 
-- The newest SDK Tools and SDK Platform-Tools packages. These packages include the Android Debugging Bridge (ADB) client and other tools that interface with the Android platform.
+- The newest SDK Tools and SDK Platform-Tools packages. These packages include the Android Debugging Bridge (ADB) client and other tools that interface with the Android platform. See the previous chapter for installation instructions.
```

Comment:
The structure of the book is different, both RE chapters come at the end of the iOS and Android chapter. See also https://github.com/OWASP/owasp-mstg/blob/master/Document/SUMMARY.md

Reply:
oops, I got confused because they follow each other in number 5b -> 5c. It's now corrected.